### ISSUE TITLE
Filter breadcrumbs on mobile need work

### ISSUE THREAD

**Issue by knolleary** [octobox/octobox#134]
The filter breadcrumbs don't behave well on small screens.

They need to either collapse down or hide completely.

Will send a pr later unless someone else wants to jump on it.

**Comment by briannelson** [comment]
@knolleary I like the breadcrumbs! One thing I noticed, and maybe this is what you were getting at with the discussion in #115, is that if you are in the Archive and apply a filter, and then remove it, you get brought back to the Inbox and not just the unfiltered archive. Is that working as expected or should I create a separate issue for that?

**Comment by andrew** [comment]
Just needs a little spacing when they are pushed onto two lines: 
![screen shot 2016-12-23 at 10 43 00 am](https://cloud.githubusercontent.com/assets/1060/21452133/a175d5ba-c8fc-11e6-876c-710eca74879a.png)

**Comment by knolleary** [comment]
@briannelson definite bug. will have a pr for it shortly.

@andrew yes, that's the easy bit. It's a bit messier when the filter's themselves spill over into two lines.

**Comment by briannelson** [comment]
@knolleary do you think this is solved by having a search box with textual filters (like the one that's being started in #164 ?

Does that search box negate the need for the breadcrumb filters? Similarly, could the sidebar filters (now that it's a bit easier to tell which ones are on) be toggles instead of just an "on" switch?

**Comment by andrew** [comment]
This is still pretty bad 🙈 
![img_fbd9fc473f03-1](https://user-images.githubusercontent.com/1060/43728406-e5a289b4-999c-11e8-8b7d-d8bdb931359c.jpeg)

**Comment by andrew** [comment]
One potential fix might be replacing the filters with search box once #742 is merged
